io.github.eclab.edisyn.desktop

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env xdg-open
+[Desktop Entry]
+Type=Application
+Name=Edisyn
+GenericName=Synthesizer patch editor and librarian
+Comment=Synthesizer Patch Editor and Librarian
+Exec=/app/bin/run.sh
```

Comment:
this is NEVER a full path.

Reply:
Ok, I have to dig deeper into this. If I only state the name of the executable - in this case "run.sh (=command in the manifest)"- the app will not be launched.

Reply:
see https://github.com/flathub/flathub/pull/5762/files#r1823404527

Reply:
fixed - /app/bin was not in PATH